server/src/repositories/asset.repository.ts

```diff
@@ -207,6 +208,7 @@ export class AssetRepository implements IAssetRepository {
     let builder = this.repository.createQueryBuilder('asset');
     builder = searchAssetBuilder(builder, options);
     builder.orderBy('asset.createdAt', options.orderDirection ?? 'ASC');
+    builder.leftJoin('asset.smartSearch', 'smartSearch');
```

Comment:
I think the intent behind this inner join was to only queue assets with smart search embeddings. It doesn't do anything as a left join so this can be removed.